rdt/hyper_transformer.py

```diff
@@ -205,7 +211,19 @@ def _validate_config(config):
 
         sdtypes = config['sdtypes']
         transformers = config['transformers']
-        if set(sdtypes.keys()) != set(transformers.keys()):
+
+        sdtype_keys = sdtypes.keys()
```

Comment:
You can't have a dict with repeated keys, so I don't think the `is_sdtype_keys_unique` check if necessary.